### ISSUE TITLE
Anisble: use SMTP instead of Postfix (sendmail)

### ISSUE THREAD

**Issue by inghamn** [bobintetley/asm3#1301]
The City of Bloomington ran into performance and spam issues when relying on Postfix for sending email out.  Most of the citizen facing forms result in an email, and these can get spammed by bots (malicious or not).  Using an SMTP server to validate and send emails relieves the ASM server of this workload.

The ansible script is currently hard-coded to set up Postfix during deployment.  It should be updated to support the configuration of SMTP.  It might even be best to hard code the ansible deployment to always use SMTP...open to ideas on this...

**Comment by bobintetley** [comment]
When you say Postfix vs SMTP I'm not sure I understand? Postfix is mail transport agent, SMTP is one of the protocols it supports. Are you talking about the difference between using sendmail vs a remote SMTP relay or something?

**Comment by inghamn** [comment]
Yes, that's right.  Instead of trying to manage postfix (sendmail) on the local server, we are switching to having ASM use our SMTP server.
